user/docs/faq.md

```diff
@@ -0,0 +1,154 @@
+---
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Troubleshooting & FAQ
+
+This page lists some common problems when installing or running NeoForge modpacks. If your problem isn't covered here, please see the [Getting Support][support] section.
+
+## Installing Java or NeoForge
+
+### I downloaded the installer file, but I have no way of running it!
+
+Please see the [Installing Java][installjava] section. Yes, even if you already have Minecraft: Java Edition installed.
+
+### I installed Java, but I'm still getting the wrong version!
+
+This likely means that your PATH variable is not set, or set incorrectly. To do this, do the following depending on your operating system:
```

Comment:
Should we explain how the `PATH` works simply, that it basically just searches for the first matching file.